themes/salix/assets/js/script.js

```diff
@@ -0,0 +1,32 @@
+const $ = str => document.querySelector(str);
```

Comment:
I did like this aspect of the design, but personally I think if we can't do it without scripting we probably shouldn't do it. There be dragons (and a less accessible, more likely to be buggy, harder to update web). I wouldn't feel strongly enough to try and block on this point though.

If we do decide to merge this, I don't feel that I know JavaScript or web things well enough to review this file, someone else might want to take a look.

Reply:
>  if we can't do it without scripting we probably shouldn't do it

I think this feature falls safely within the realm of progressive enhancement. If you don't run the JS it just leaves all the headers underlined. Perhaps we can chat elsewhere about the roll of JS in the browser.